src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers/CommandHandlers/ProviderApproveCohortCommandHandler.cs

```diff
@@ -24,13 +24,27 @@ public class ProviderApproveCohortCommandHandler : IHandleMessages<ProviderAppro
 
         public async Task Handle(ProviderApproveCohortCommand message, IMessageHandlerContext context)
         {
-            _logger.LogInformation($"Handled {nameof(ProviderApproveCohortCommand)} with MessageId '{context.MessageId}'");
+            try
+            {
+                _logger.LogInformation($"Handled {nameof(ProviderApproveCohortCommand)} with MessageId '{context.MessageId}'");
```

Comment:
I know this is not your change, but should this be at the end when the message has been handled, or the log message needs to change? 
But not that worried about it, may be it is there to just identify the message is received. 
I will leave it to you to decide whether this should be changed.